packages/core/src/ui-helpers.ts

```diff
@@ -0,0 +1,4 @@
+export { StoredDeploymentSerializer } from "./new-api/stored-deployment-serializer";
+export * from "./new-api/types/module";
+export { FutureType } from "./new-api/types/module";
```

Comment:
Isn't this already exported in the line above?

```suggestion
```

Reply:
Good catch. Thanks.